src/bin/bat/clap_app.rs

```diff
@@ -398,6 +398,7 @@ pub fn build_app(interactive_output: bool) -> ClapApp<'static, 'static> {
                         !&[
                             "auto",
                             "full",
+                            "default",
```

Comment:
Oh I see one thing that is missing now; updating the --help text in src/bin/bat/clap_app.rs line 425-ish

Reply:
I've added those, and also header-filename and header-filesize in the short help description.